edivorce/apps/core/templatetags/format_utils.py

```diff
@@ -21,7 +21,7 @@ def linebreaksli(value):
     value = re.sub(r'\r\n|\r|\n', '\n', value.strip())  # normalize newlines
     lines = re.split('\n', value)
     lines = ['<li>%s</li>' % line for line in lines if line and not line.isspace()]
-    return mark_safe('\n'.join(lines))
+    return mark_safe('\n'.join(lines)) # nosec
```

Comment:
Just confirming that **linebreaksli** is only called on PDF forms (form1, form38, form52), and these are called through Weasyprint.  
Also we always call striptags before we call linebreaksli, so this is safe